ui/src/app/job-list/table/table.component.html

```diff
@@ -64,30 +52,30 @@
                   <span>Timing Diagram</span>
                 </a>
               </mat-list-item>
-              <mat-list-item *ngIf="j.name">
-                  <clr-tooltip>
-                    <a (click)="filterOnFieldValue('name', j.name)" clrTooltipTrigger>
-                      <clr-icon shape="filter-2" size="24"></clr-icon>
-                      <span class="job-title-filter">Filter by name:{{ j.name }}</span>
-                      <clr-tooltip-content clrPosition="top-right" clrSize="xs" *clrIfOpen>
-                        <span>Filter by name:{{ j.name }}</span>
-                      </clr-tooltip-content>
-                    </a>
-                  </clr-tooltip>
-              </mat-list-item>
+              <mat-list-item *ngIf="j[displayedColumns[1]] && canFilterBy(displayedColumns[1])">
+                <clr-tooltip>
+                  <a (click)="filterOnFieldValue(displayedColumns[1], j[displayedColumns[1]])" clrTooltipTrigger class="inline-filter">
+                    <clr-icon shape="filter-2" size="24"></clr-icon>
+                    <span>Filter by {{ displayedColumns[1] }}:{{ j[displayedColumns[1]] }}</span>
+                    <clr-tooltip-content clrPosition="top-right" clrSize="xs" *clrIfOpen>
+                      <span>Filter by {{ displayedColumns[1] }}:{{ j[displayedColumns[1]] }}</span>
+                    </clr-tooltip-content>
+                  </a>
+                </clr-tooltip>
+            </mat-list-item>
             </mat-list>
           </mat-card-actions>
         </mat-card>
       </mat-menu>
     </mat-cell>
   </ng-container>
 
-  <!-- Implementation-specific additional columns -->
+  <!-- Implementation-specific columns -->
   <ng-container *ngFor="let df of displayFields">
     <ng-container matColumnDef="{{ df.field }}">
       <mat-header-cell *matHeaderCellDef>{{ df.display }}</mat-header-cell>
       <mat-cell class="additional-column" *matCellDef="let j">
-        <div class="display-field" *ngIf=canEdit(df)>
+        <div class="display-field" *ngIf="canEdit(df) || (canFilterBy(df.field) && (getFieldValue(j, df) != ''))">
```

Comment:
tiny nit: I think the extra parens around `(getFieldValue(j, df) != '')` are unecessary:
```
... || (canFilterBy(df.field) && getFieldValue(j, df) != '')
```

Reply:
good point – simplified